src/kernels/jupyter/launcher/jupyterNotebookProvider.ts

```diff
@@ -2,52 +2,15 @@
 // Licensed under the MIT License.
 
 import { inject, injectable } from 'inversify';
-import {
-    ConnectNotebookProviderOptions,
-    GetServerOptions,
-    IJupyterConnection,
-    IKernelConnectionSession,
-    isLocalConnection,
-    NotebookCreationOptions
-} from '../../types';
-import { Cancellation } from '../../../platform/common/cancellation';
+import { ConnectNotebookProviderOptions, IJupyterConnection } from '../../types';
 import { IJupyterNotebookProvider, IJupyterServerProvider } from '../types';
 
 // When the NotebookProvider looks to create a notebook it uses this class to create a Jupyter notebook
 @injectable()
 export class JupyterNotebookProvider implements IJupyterNotebookProvider {
```

Comment:
Could other classes just use the ServerProvider directly so that this class can be dropped?